goto-line-preview.el

```diff
@@ -48,6 +48,16 @@
   :group 'goto-line-preview
   :type 'hook)
 
+(defcustom goto-line-preview-hl-duration 1
+  "Duration of highlight when change preview line."
+  :group 'goto-line-preview
+  :type 'integer)
```

Comment:
I recommend this accept `nil`, and no highlighting when it's `nil`.

Reply:
Thanks for your suggestion, you are right. I edit the code, now it can accept nil, is it better now?

Reply:
Thank you!